tests/test-util/test.logger.js

```diff
@@ -65,36 +72,42 @@ describe('logger', () => {
     it('does not log debug packets unless verbose', () => {
       const log = new ConsoleStream({verbose: false});
       const localProcess = fakeProcess();
+      // $FLOW_IGNORE: fake process for testing reasons.
```

Comment:
could this also be solved with an [interface](https://flowtype.org/docs/quick-reference.html#interfaces)?